DoePaAdminViewModel/Services/DoePaAdminService.cs

```diff
@@ -108,6 +110,35 @@ public async Task<Anstellungsdetail> CreateAnstellungsdetailAsync(CancellationTo
             return await AddDataToDbSetAsync(DBContext.Anstellungsdetails, cancellationToken);
         }
 
+        public async Task<IEnumerable<EmployeeInvoicedHours>> GetEmployeeInvoicedHoursAsync(string email, DateTime from, DateTime to, CancellationToken cancellationToken = default)
+        {
+            var result = DBContext
+                .Ausgangsrechnungspositionen
+                        .Include(arp => arp.ZugehoerigeRechnung)
+                        .Include(arp => arp.ZugehoerigeKostenstelle)
+                        .Include(arp => arp.ZugehoerigeAuftragsposition)
+                            .ThenInclude(zap => zap.Auftrag)
+                                .ThenInclude(a => a.ZugehoerigesProjekt)
+                                    .ThenInclude(p => p.Rechnungsempfaenger)
+                                        .ThenInclude(r => r.ZugehoerigerKunde)
+                        .Include(arp => arp.ZugehoerigeAuftragsposition.Auftrag.VerantwortlicherMitarbeiter)
+                        .Include(arp => arp.ZugehoerigeAbrechnungseinheit)
+                    .Where(arp => arp.ZugehoerigeAuftragsposition.Auftrag.VerantwortlicherMitarbeiter.Email == email)
+                    .Where(arp => arp.ZugehoerigeAbrechnungseinheit.Name.Equals("Stunden"))
```

Comment:
This is okay for testing purposes. But we can not rely on having the Abrechnungseinheit "Stunden" be created in any environment. Now that I think about: We should change the whole thing and return the Abrechnungseinheit as part of the apis resultset. So the frontend can display the Abrechnungseinheit to the user. I will create an issue for this.

Reply:
Yes, I had already thought about that :)